consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftRound.java

```diff
@@ -131,6 +131,27 @@ public ConsensusRoundIdentifier getRoundIdentifier() {
     return roundState.getRoundIdentifier();
   }
 
+  /**
+   * Create a block
+   *
+   * @param headerTimeStampSeconds of the block
+   * @return a Block
+   */
+  public Block createBlock(final long headerTimeStampSeconds) {
+    LOG.debug("Creating proposed block. round={}", roundState.getRoundIdentifier());
+    return blockCreator.createBlock(headerTimeStampSeconds, this.parentHeader).getBlock();
+  }
+
+  /**
+   * Send proposal message.
+   *
+   * @param block to send
+   */
+  public void sendProposalMessage(final Block block) {
+    LOG.trace("Creating proposed block blockHeader={}", block.getHeader());
+    updateStateWithProposalAndTransmit(block);
+  }
+
```

Comment:
```suggestion
```

Reply:
OK, I understand those changes now, but I think I did this with some purpose, I have to review this too...

Reply:
OK, I remember I did this because there is a _createAndSendProposalMessage_ so I thougth creating the _sendProposalMessage_ will be more clear, because we would have this one as a contrapart to the other, like

```
createBlock + sendProposalMessage = createAndSendProposalMessage
```

But I'll delete it anyway, actually it seems that createAndSendProposalMessage is only used from unit tests, so it may be deleted as well.

Reply:
I remmeber now why I did that, since there was a `createAndSendProposalMessage ` I actually break it in two, so I created

```
createBlock
sendProposalMessage
```

where `createBlock + sendProposalMessage = createAndSendProposalMessage`, so it have some kind of simetry.

But now I checked and `createAndSendProposalMessage` is not used anywhere except on 4 tests in QbftRoundTest, so I think we can delete both methods ands tests, what do you think?

The only thing I don't like is that IBFT has that `createAndSendProposalMessage`, it seems IBFT and QBFT are mirror implemenations, and this will break that simmetry... (yes, I suffer a liitle from OCD...).

Anyway, I commented both `createAndSendProposalMessage `and the tests for you to review...